src/methods/to.js

```diff
@@ -441,8 +441,20 @@ export class ToImplementation {
       default:
         /*eslint-enable */
         if (realm.isInPureScope()) {
-          // will be serialized as Object.assign(serialized_arg)
-          obj = AbstractValue.createFromType(realm, ObjectValue, "explicit conversion to object", [arg]);
+          // get the global Object.assign
+          let globalObj = Get(realm, realm.$GlobalObject, "Object");
+          invariant(globalObj instanceof ObjectValue);
+          let objAssign = Get(realm, globalObj, "assign");
+
+          obj = AbstractValue.createTemporalFromBuildFunction(
+            realm,
+            ObjectValue,
+            [objAssign, arg],
+            ([objAssignNode, argNode]) => t.callExpression(objAssignNode, [argNode]),
+            {
+              kind: "explicit conversion to object",
```

Comment:
If I make a typo in this string on master, then `ToObject.js` serializer test from #1690 fails.

But if I make a typo in this branch, I don't see failing tests. Does this mean this `kind` is now unused? Why? Can/should it be removed?

Reply:
That's because I removed this part in this PR: https://github.com/facebook/prepack/pull/1690/files#diff-35328c9e9230682947d04cd307cec77cR1653

Reply:
Yes but that's not the only place in the codebase checking for that string. It means we don't cover those other places by tests.

Reply:
We should aim to, which means the original tests for this feature didn't add all the coverage they should have. I also tried renaming it to see if tests broke when I made this PR and thought it was odd.